### PR TITLE
Bump quote to 0.4

### PR DESCRIPTION
**Pull request by Eijebong**

### CODE REVIEW

File: tests/expectations/tests/objc_whitelist.rs [hunk 1]
```diff
@@ -18,7 +17,7 @@ impl protocol_SomeProtocol for id {
     }
     unsafe fn protocolClassMethod() {
         msg_send!(
-            objc::runtime::Class::get("SomeProtocol").expect("Couldn't find SomeProtocol"),
+            objc::runtime::Class::get("SomeProtocol").expect("Couldn\'t find SomeProtocol"),
```
Comment (fitzgen) [inline tests/expectations/tests/objc_whitelist.rs]: Looks like there is too much escaping happening now.

Comment (Eijebong) [reply tests/expectations/tests/objc_whitelist.rs]: Nothing I can do about it unfortunately (unless I missed something)

File: src/codegen/mod.rs [hunk 2]
```diff
@@ -3290,7 +3286,7 @@ fn objc_method_codegen(
         let class_name = class_name
             .expect("Generating a class method without class name?")
             .to_owned();
-        let expect_msg = format!("Couldn't find {}", class_name);
+        let expect_msg = proc_macro2::Literal::string(&format!("Couldn't find {}", class_name));
```
Comment (fitzgen) [inline src/codegen/mod.rs]: HUh, I guess `proc_macro2` isn't smart about string escaping.

Comment (Eijebong) [reply src/codegen/mod.rs]: No, https://github.com/alexcrichton/proc-macro2/blob/master/src/stable.rs#L576

Comment (fitzgen) [reply src/codegen/mod.rs]: Maybe file a bug upstream?

Comment (Eijebong) [reply src/codegen/mod.rs]: https://github.com/alexcrichton/proc-macro2/issues/60

**Pull request merged**